delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/PointerName.html

```diff
@@ -2,12 +2,37 @@ <h2>Why is this an issue?</h2>
 <p>
   Code that follows a consistent naming convention is self-documenting. In Delphi, all pointer names
   should be named identically to the dereferenced type, with the type prefix swapped for
-  <code>P</code>.
+  <code>P</code>, case original prefix are <code>T</code>, <code>E</code> or <code>I</code>.
```

Comment:
The end of this sentence doesn't quite make sense - it's probably okay to remove, since this information is described later on.

```suggestion
  <code>P</code>.
```
Rewording is also OK if you would prefer.